### DISCUSSION TITLE
Issues with 0.15.0

### DISCUSSION THREAD

**Discussion by lukedays** [cunnane/xloil#14] [General]
Hi Steven! I did a fresh install of 0.15, but the ribbon didn't appear. Do I need to activate something?
Also, the auto reloading of modules didn't work as well (even those with the same name as the workbook). I tried changing just the code inside the decorated function as instructed in the docs.
Thanks for looking into the issues!!

**Comment by cunnane** [comment]
Do you have "_xloil.xloil_ribbon_" in the _LoadModules_ in directive in your ini file?  You'd need to delete your `%APPDATA%\xlOil\xlOil.ini` file before installation for the installer to update it.  Is there anything in the logs that looks like an error?
